### PR TITLE
fix test logging

### PR DESCRIPTION
**Pull request by lindsaylandry**
test steps are recorded again and are visible in the HTML report

### CODE REVIEW

File: lib/chemistrykit/cli/cli.rb [hunk 1]
```diff
@@ -126,7 +126,8 @@ def rspec_config(config)
         end
 
         ::RSpec.configure do |c|
-          c.capture_log_messages
+          log = Logging.logger['test steps']
+          c.capture_log_messages
```
Comment (jaysirju) [inline lib/chemistrykit/cli/cli.rb]: does it work with this line?  This was supposed to output to a file at the end of the test, but you're doing that explicitly by adding an Appender now.

File: Gemfile [hunk 2]
```diff
@@ -21,7 +21,7 @@ gem 'parallel_split_test', git: 'https://github.com/animoto/parallel_split_test.
 gem 'allure-rspec', git: 'https://github.com/animoto/allure-rspec.git', branch: 'rspec-2'
 gem 'compass'
 gem 'zurb-foundation'
-gem 'logging'
+gem 'logging', '2.0.0'
```
Comment (jaysirju) [inline Gemfile]: Hmmm, I'm probably at fault for this, but is there a reason why this is both in the Gemfile and Gemspec?

**Comment by jaysirju** [comment]
I've mentioned a couple clarifying questions, but looks great otherwise.

File: Gemfile (hunk 2 above)
Comment (lindsaylandry) [inline Gemfile]: not sure, will this be ok if it's just in the gemspec?

File: lib/chemistrykit/cli/cli.rb (hunk 1 above)
Comment (lindsaylandry) [inline lib/chemistrykit/cli/cli.rb]: i'm not entirely sure how it works, but it does. Possibly, Logger.logger['test steps'] is a pointer to a variable that i can add appenders to in the cli script, and then calling the same thing in test_common_library points back to that variable that now has appenders on it.

Like i said, this is just a guess, but this made it work.

**Pull request closed without merging**